src/Scrabble/Move/Move.hs

```diff
@@ -42,9 +42,13 @@ createMove' :: Validator -- ^
 createMove' validate b (Rack rack) wp dict = if valid then go else errMsg where
   errMsg        = Left "error: rack missing input letters"
   rackLetters   = fmap letter rack
-  valid         = containsAll (toString putLetters) (toString rackLetters)
+  valid         = containsAllWithBlanks (toString putLetters) (toString rackLetters)
+  blanksUsed    = countBlanksUsed (toString putLetters) (toString rackLetters)
   putLetters    = letter <$> wordPutTiles wp
-  rackRemainder = fmap fromLetter $ foldl' (flip delete) rackLetters putLetters
+  rackRemainder = fmap fromLetter $
+                  applyNTimes blanksUsed (delete Blank) $
+                  foldl' (flip delete) rackLetters putLetters
+  applyNTimes n f = foldr (.) id (replicate n f)
```

Comment:
We should change it back. If someone places a blank and uses Q, and then you take the Q out of their rack, that is a huge problem. They would get a potentially really bad score for their Q, and wouldn't be able to use it later. We have to assume they know what they are doing.